src/etl/tissue_enrichment/tissue_enrichment.py

```diff
@@ -0,0 +1,195 @@
+"""Calculation of tissue enrichment!"""
+
+from __future__ import annotations
+
+# import numpy
+import math
+from typing import TYPE_CHECKING
+
+import pyspark.sql.functions as f
+import pyspark.sql.types as t
+from pyspark.sql.types import FloatType
+from pyspark.sql.window import Window
+from scipy import special as sps
+
+if TYPE_CHECKING:
+    from pyspark.ml.linalg import VectorUDT
+    from pyspark.sql import DataFrame
+
+
+def _melt(
```

Comment:
I would recommend to add an example doctest here. Every steps seems to be reasonable, however it is hard to follow what is exactly happening.